### ISSUE TITLE
Cannot gain access code (token) via for SharePoint on duck(linux)

### ISSUE THREAD

**Issue by junukwon7** [iterate-ch/cyberduck#15587]
**Describe the bug**
Cannot log into sharepoint

**To Reproduce**
1. Install duck on ubuntu 22.04
2. Do duck --username username --list sharepoint:/
3. The prompt says to open web browser to authenticate and obtain an authorization code, with a login.microsoftonline URL.
4. Using my mac (ubuntu-server is headless), log-in.
5. Approve the app on behalf the organization (I'm the admin).
6. Nothing happens, stuck here.

Additionally, with development tools present in Chromium-based browsers, the error says 

```text
Failed to launch 'x-cyberduck-action://oauth/?code=0.AcoA3RBTXEtpLUqsidjWsrNQJo_BC_QCzRJCt_EVJD5....shVLs&state=ht....dh&session_state=cf2e58.....44cf' because the scheme does not have a registered handler.
```

Tried copy-pasting the code="" part, but didn't work.

**Expected behavior**

**Screenshots**
Basically, stuck here. Clicking the continue button triggers the action, but not showing the code.
![image](https://github.com/iterate-ch/cyberduck/assets/48399106/394fc70c-f10e-43ae-b0aa-2cbb0a6dbdd7)

**Desktop (please complete the following information):**
 - OS: ubuntu-server 22.04 as a PVE LXC
 - Version: Cyberduck 8.7.2 (40975).

**Additional context**
As a software dev with some experience for Microsoft SSO integration on products, I guess the return address and/or the App Registration settings should be changed. Basically it should simply print out the code visually, as in some cases the machine used for browser verification might not have a duck executable.

**Comment by dkocher** [comment]
I can confirm the custom redirect URI is not registered with Cyberduck CLI and is unable retrieve events from the web browser. Duplicate for #13587. As a workaround use the connection profile from [^1]

[^1]: iterate-ch/profiles#77
